sdk/core/System.ClientModel/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.targets))\Directory.Build.targets" />
```

Comment:
Unless I'm reading something wrong, I think this will look for a file `Directory.Build.targets` in `$/sdk`, which does not exist.   I think you may need to adjust the path to move up one more directory.